app/controllers/concerns/idv_session.rb

```diff
@@ -48,6 +48,10 @@ def idv_question_number=(num)
     idv_session[:question_number] = num
   end
 
+  def idv_params=(params)
+    idv_session[:params] = params
+  end
+
```

Comment:
I think we should rename this argument since it's already reserved in controllers